src/test/frontend/replicasetlist/replicasetlist_controller_test.js

```diff
@@ -0,0 +1,45 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ReplicaSetListController from 'replicasetlist/replicasetlist_controller';
+import replicaSetListModule from 'replicasetlist/replicasetlist_module';
+
+describe('Replica set list controller', () => {
+  /**
+   * @type {!ReplicaSetCardController}
+   */
+  let ctrl;
+  /** @type {!ui.router.$state} */
+  let state;
+
+  beforeEach(() => {
+    angular.mock.module(replicaSetListModule.name);
+
+    angular.mock.inject(($controller, $state) => {
+      state = $state;
+      ctrl = $controller(ReplicaSetListController, {replicaSets: {replicaSets: []}});
+    });
+  });
+
+  it('should go to deploy page', () => {
```

Comment:
hmm...I would suggest to add more description. Something like:


+  it('should provide a floating FAB button to add new element', () => {
 +    // given the ... page loaded
 +    spyOn(state, 'go');
 +
 +    // when clicking on button
 +    ctrl.toDeploy();
 +
 +    // then browser opens the page to deploy new app
 +    expect(state.go).toHaveBeenCalledWith('deploy');
 +  });